app/controllers/artists_controller.rb

```diff
@@ -5,6 +5,7 @@ def index
 
   def show
     @artist = Artist.find(params[:id])
+    @buyer = current_buyer
```

Comment:
We didn't do this earlier, but at the top of each react component, we should be writing a comment that details the props that are passed into the component. see the BuyerProfile react component for an example